src/site/content/en/blog/top-cwv-2023/index.md

```diff
@@ -0,0 +1,203 @@
+
+Sometimes it's not possible to know the exact size of dynamic content since it is, by its very nature, dynamic. However, even if you don't know the exact size, you can still take steps to reduce the severity of layout shifts. **Setting a sensible `min-height`** is almost always better than allowing the browser to use the default height of `0px` for an empty element. Using a `min-height` is also usually an easy fix as it still allows the container to grow to the final content height if needed—it has just reduced that amount of growth from the full amount to a hopefully more tolerable level.
+
+### Ensure pages are eligible for bfcache
+
+Browsers use a navigation mechanism called the [back/forward cache](/bfcache/)—or bfcache for short—to instantly load a page from earlier or later in the browser history directly from a memory snapshot.
+
+The bfcache is a significant browser-level performance optimization, and it entirely eliminates the layout shifts during page load, which for many sites is where most of their CLS occurs. The introduction of the bfcache caused [the biggest improvement in CLS](https://twitter.com/anniesullie/status/1491399685961293828?s=20&t=k7JgTjdO21uMpeOuOofroA) that we saw in 2022.
+
+Despite this, [a significant number of websites](https://almanac.httparchive.org/en/2022/performance#bfcache-eligibility) are ineligible for the bfcache and so are missing out on this free web performance win for a significant number of navigations. Unless your page is loading sensitive information that you don't want to be restored from memory, you'll want to make sure that your pages are eligible.
+
+Site owners should check that their pages are [eligible for the bfcache](/bfcache/#optimize-your-pages-for-bfcache) and work on any reasons why they are not. Chrome already [has a bfcache tester in DevTools](/bfcache/#test-to-ensure-your-pages-are-cacheable) and this year we plan to enhance tooling here with [a new Lighthouse audit performing a similar test](https://github.com/GoogleChrome/lighthouse/issues/13960) and [an API to measure this in the field](https://chromestatus.com/feature/5684908759449600).
+
+While we have included the bfcache in the CLS section, as we saw the biggest gains there so far, the bfcache will also improve the other Core Web Vitals, i.e. with 0 LCP and much improved FID and INP. It is one of [a number of instant navigations](https://calendar.perfplanet.com/2022/fast-is-good-instant-is-better/) available to drastically improve page navigations.
```

Comment:
```suggestion
While we have included the bfcache in the CLS section, as we saw the biggest gains there so far, the bfcache will generally also improve other Core Web Vitals too. It is one of [a number of instant navigations](https://calendar.perfplanet.com/2022/fast-is-good-instant-is-better/) available to drastically improve page navigations.
```

Good spot, and I think this is actually inaccurate anyway since LCP will not be 0 and I don't think we have any evidence to show that bfcache improves INP.

Reply:
Done